src/main/sidebar/sidebar.component.scss

```diff
@@ -191,3 +179,10 @@
 .plus {
   width: 1.0625em;
 }
+.background-color{
+  background-color: #2A67F4;
+  border-radius: 5px;
+  cursor: default;
+
```

Comment:
after creating PR or commit your changes please take a look at your change files and see if there is any extra space then please remove it